processor/src/main/resources/org/mapstruct/ap/internal/model/NestedPropertyMappingMethod.ftl

```diff
@@ -19,7 +19,7 @@
      limitations under the License.
 
 -->
-<#lt>private <@includeModel object=returnType/> ${name}(<#list parameters as param><@includeModel object=param/><#if param_has_next>, </#if></#list>) {
+<#lt>private <@includeModel object=returnType/> ${name}(<#list parameters as param><@includeModel object=param/><#if param_has_next>, </#if></#list>) <@throws/> {
```

Comment:
The generated code fails because you have `) <@throws/> {` (mind the whitespace between `)` and `<@throws`. If you replace that with `)<@throws/> {` then those tests won't fail. Have a look how it is in the `BeanMappingMethod.ftl`